Core.lua

```diff
@@ -39,11 +39,18 @@ local LSMUpdateTimer
 ---------------------------------------------------------------------------------------------------
 
 if Addon.WOW_USES_CLASSIC_NAMEPLATES then
-  local function CalculateSynchedNameplateSize()
+  local function CalculateSynchedNameplateSize(friendlyUnit)
     local db = Addon.db.profile.settings
   
-    local width = db.healthbar.width
-    local height = db.healthbar.height
+    local width, height
+    if friendlyUnit then
+      width = db.healthbar.widthFriendly
+      height = db.healthbar.heightFriendly
+    else
+      width = db.healthbar.width
+      height = db.healthbar.height
+    end
+        
```

Comment:
I think I changed this code in a wrong way. In Classic, it's not possible to have separate nameplates sizes for friendly and enemy plates as the clickable area is always the same for both. This should be part of the non-classic code. I need to fix this afterwards.